tbconnect/tasks.py

```diff
@@ -153,10 +153,12 @@ def send_tbcheck_data_to_cci(data):
         if (
             response.status_code == 200
             and b'"Received Successfully"' == response.content
+            or response.status_code == 200
+            and b'"Received Sucessfully"' == response.content
```

Comment:
Is it sometimes spelled correctly and sometimes not?

Reply:
I once received the correct spelled but now i'm getting the missed spelled on.

Reply:
I will stick to the latest one that i received